operations/mimir-mixin/alerts/blocks.libsonnet

```diff
@@ -142,10 +142,10 @@
           alert: $.alertName('IngesterTSDBWALCorrupted'),
           expr: |||
             # alert when there are more than one corruptions
-            count by (%(alert_aggregation_labels)s) (rate(cortex_ingester_tsdb_wal_corruptions_total[%(range_interval)s]) > 0) > 1
+            count by (%(alert_aggregation_labels)s, %(per_instance_label)s) (rate(cortex_ingester_tsdb_wal_corruptions_total[%(range_interval)s]) > 0) > 1
             and
             # and there is only one zone
-            count by (%(alert_aggregation_labels)s) (group by (%(alert_aggregation_labels)s, %(per_job_label)s) (cortex_ingester_tsdb_wal_corruptions_total)) == 1
+            count by (%(alert_aggregation_labels)s, %(per_instance_label)s) (group by (%(alert_aggregation_labels)s, %(per_instance_label)s, %(per_job_label)s) (cortex_ingester_tsdb_wal_corruptions_total)) == 1
```

Comment:
this is meant to count the number of zones, not the number of pods. I think here the message should be corrected instead of the query